src/test/java/com/woowacamp/soolsool/core/liquor/domain/vo/LiquorAlcoholTest.java

```diff
@@ -18,20 +18,18 @@ void create() {
         double alcohol = 17.2;
 
         /* when & then */
-        assertThatCode(() -> new LiquorAlcohol(alcohol))
-            .doesNotThrowAnyException();
+        assertThatCode(() -> new LiquorAlcohol(alcohol)).doesNotThrowAnyException();
     }
 
     @Test
-    @DisplayName("술 도수가 0 이상이 아닌 경우 ShoppingException을 던진다.")
+    @DisplayName("술 도수가 0 이상이 아닌 경우 SoolSoolException을 던진다.")
     void createFailWithIsNotGreaterThanZero() {
         /* given */
 
 
         /* when & then */
-        assertThatCode(() -> new LiquorAlcohol(-1.2))
-            .isExactlyInstanceOf(ShoppingException.class)
-            .hasMessage("술 도수는 0 이상 실수여야 합니다.");
+        assertThatCode(() -> new LiquorAlcohol(-1.2)).isExactlyInstanceOf(SoolSoolException.class)
+                .hasMessage("술 도수는 0 이상 실수여야 합니다.");
```

Comment:
한 줄에 하나씩 쓰는 건 어뗘요????

```suggestion
        assertThatCode(() -> new LiquorAlcohol(-1.2))
            .isExactlyInstanceOf(SoolSoolException.class)
            .hasMessage("술 도수는 0 이상 실수여야 합니다.");
```

Reply:
종하요